corporate/views/remote_billing_page.py

```diff
@@ -154,7 +171,7 @@ def remote_realm_billing_finalize_login(
         context = {
             "remote_server_uuid": remote_server_uuid,
             "remote_realm_uuid": remote_realm_uuid,
-            "remote_realm_host": remote_realm.host,
+            "host": remote_realm.host,
```

Comment:
What's the thinking around this rename?

Reply:
This is the template that's "shamelessly re-used" as I called it, by the new flow. So where `remote_realm_host` used to go on the HTML page, now the `remote_server.hostname` goes in that flow. So renamed this template variable to something more general.

The template needs more generalizing in its naming as I mentioned, because it's file name and HTML element classes etc. are still named for `remote-realm`

Reply:
I see, OK.